content/pages/meetup/2024/0904-nycu.rst

```diff
@@ -0,0 +1,123 @@
+========================================
+Meetup 2024 September 4th in NYCU
+========================================
+
+:date: 2024-08-27 15:58
+:url: meetup/2024/0904-nycu
+:save_as: meetup/2024/0904-nycu.html
+
+Date
+-----
+
+* Date: September 4th, 2024
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+There will be many different subjects discussed at the same time.
+
+* 18:30-19:30 Free discussion
+* 19:30-21:30 Subjects discussion
+
+|
+
+Subjects
+------------------
+
+modmesh development discussion
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+In this Meetup, `Chun-Shih <https://github.com/ThreeMonth03?tab=followers>`__ 
+will lead the discussion on the `modmesh project <https://github.com/solvcon/modmesh>`__. 
+Through participating in the modmesh project discussion, you can develop the following skills:
+
+* Learn how to communicate and contribute to open-source projects.
+* Get familiar with package management and script writing in the development environment, 
+  such as Qt6 and PySide.
+* Learn to write modern C++, Python, and unit tests.
+
+Additionally, this project is very beginner-friendly. 
+For those who have participated fewer than 4 times, 
+we have prepared a series of modmesh development tutorials, which include:
+
+1. Setting up and demonstrating modmesh.
+2. Basics of Git and GitHub.
+3. Writing Pybind11 and Unittest.
+4. Fundamentals of CMake, GNU Make, and GitHub Workflow.
+
+Leetcode practice discussion
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+In this Meetup, `Kuan-Wei <https://github.com/visitorckw>`__ will lead the LeetCode study group. 
+During the LeetCode discussions, we won't just focus on solving problems; 
+there will be more in-depth technical discussions and communication practice, 
+along with training on writing clean code. 
+Through this activity, we aim to develop the following skills:
+
+* A strong foundation in data structures and algorithms,
+  enabling quick solutions to basic problems and follow-up questions.
+* The ability to clearly express and present clean code,
+  making algorithms easy to understand and maintain.
+
+Prepare upcoming sciwork events
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+Additionally, we will discuss the following items to prepare the upcoming events.
+
+* Task arrangement for volunteers
+* Promotion plan for upcoming events
+
+|
+
+Sign up
+------------
+
+The meetup is free. 
+Please register on `discord event <https://discord.com/channels/730297880140578906/1007075707400237067/1275445577521565837>`__. 
```

Comment:
This links to 8/21 event.  Please correct.  (Maybe you need to create the event before making this PR.)

<img width="601" alt="image" src="https://github.com/user-attachments/assets/aef0f623-e07d-492a-b481-8bb8f89e3d6c">